Makefile

```diff
@@ -1,12 +1,24 @@
 IMAGE ?= quay.io/packit/packit-service-fedmsg:dev
-CONTAINER_ENGINE ?= docker
+CONTAINER_ENGINE ?= $(shell command -v podman 2> /dev/null || echo docker)
 ANSIBLE_PYTHON ?= /usr/bin/python3
 AP ?= ansible-playbook -vv -c local -i localhost, -e ansible_python_interpreter=$(ANSIBLE_PYTHON)
+TEST_IMAGE := fedmsg-test
 TEST_TARGET := ./tests/
+CHECK_PACKAGE ?= rpm -q --quiet
+INSTALL_PACKAGE ?= dnf install
```

Comment:
@bcrocker15 I renamed and tweaked these variables a little compared to what you've done in OGR. With `?=` it's possible to change these explicitly when calling `make`, so a user can do that if they are on an OS using some other package management tools. For Fedora Linux these would just work.

This seems less complicated than detecting the OS and deciding based on that.